docs/reference/run.mdx

```diff
@@ -510,6 +510,34 @@ mycontainer                      wget           477678            44591
 </TabItem>
 </Tabs>
 
+It's possible to hide and show some fields by prefixing them with +/-, for instance
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
+
+```bash
+$ kubectl gadget run trace_exec:latest --fields=-error,-timestamp
+COMM  PID  TID  PCOMM  PPID  ARGS  K8S.NODE  K8S.NAMESPACE  K8S.PODNAME  K8S.CONTAINERNAME  USER  LOGINUSER  GROUP
+
+$ ./kubectl-gadget run trace_exec:latest --fields=+uid,+pid
```

Comment:
kubectl gadget

Reply:
@mauriciovasquezbernal please check. Thank you 🙏

Reply:
I still see ./kubect-gadget.